src/html-renderer.js

```diff
@@ -1,29 +1,19 @@
 'use strict';
 
-const format = require('stringformat');
-
 const templates = require('./templates');
 
 module.exports = {
-  renderedComponent: function(data) {
+  renderedComponent: data => {
```

Comment:
should we start using the es6 syntax for methods instead of `=>` ?, ie:
```
{
  renderComponents(data) {...},
  unrenderedComponent(data){...}
}
```

I think we touched on this lightly some times ago, but don't recall what we decided upon

Reply:
I think I am not against it, but perhaps not here, as we mostly use arrows so it sounds like a more consistent choice for now